backoffice/common/items/Input.tsx

```diff
@@ -60,6 +62,110 @@ export const Input = <T extends FieldValues>({
   </div>
 );
 
+export const FormatNumberInput = <T extends FieldValues>({
```

Comment:
I think we can use react-number-format library instead of handling manually 
https://s-yadav.github.io/react-number-format/docs/props